imap_processing/idex/idex_l1a.py

```diff
@@ -247,13 +197,15 @@ def __init__(
         self.high_sample_trigger_time = 0
         self._set_sample_trigger_times(header_packet)
 
-        # Iterate through the trigger description dictionary and pull out the values
-        self.trigger_values = {
-            trigger.name: header_packet[trigger.packet_name].raw_value
-            for trigger in TRIGGER_DESCRIPTION_DICT.values()
+        # Iterate through every telemetry item not in the header and pull out the values
+        self.telemetry_items = {
+            key.lower(): val
+            for key, val in header_packet.items()
+            if key not in header_packet.header.keys()
```

Comment:
Glad you found a way to avoid defining that large `TRIGGER_DESCRIPTION_DICT`. Nice.